docs/schema/crawledpropertyinfo-complextype-sps15xsdsearchset2.md

```diff
@@ -1,26 +1,25 @@
 ---
 title: "CrawledPropertyInfo complexType (SPS15XSDSearchSet2)"
+description: Outlines information, a definition, and elements and attributes for the CrawledPropertyInfo complexType in Sharepoint.
 manager: arnek
-ms.date: 3/9/2015
+ms.date: 06/09/2022
 ms.audience: ITPro
 ms.topic: article
 ms.prod: sharepoint
 ms.localizationpriority: medium
 ms.assetid: 6a4d1072-03d8-3141-d6d9-6a384c2795b8
-description: "Last modified: March 09, 2015"
 ---
 
 # CrawledPropertyInfo complexType (SPS15XSDSearchSet2)
 
  
   
 ## Type information
+**Namespace**: http://schemas.datacontract.org/2004/07/Microsoft.Office.Server.Search.Administration 
```

Comment:
We've been directed to keep Markdown tables as tables. You can use "Information/Location" or "Name/Value" as headers, or if you find a more specific header that fits, use that.